settings.gradle

```diff
@@ -1 +1,3 @@
 include ':app', ':libtoolscommon'
+include ':ios'
+include ':commons'
```

Comment:
That is wrong in my environment, the project build fine but Android Studio can not see these packages, please change it to:

`include ':app', ':libtoolscommon', ':ios', ':commons'`